app/src/test/java/org/oppia/app/testing/options/AppLanguageFragmentTest.kt

```diff
@@ -45,11 +102,108 @@ class AppLanguageFragmentTest {
     }
   }
 
+  @Test
+  @Config(qualifiers = "sw600dp")
+  fun testAppLanguage_clickAppLanguage_changeAppLanguageToFrench_checkOptionsFragmentIsUpdatedCorrectly() { // ktlint-disable max-line-length
```

Comment:
Please abbreviate. We shouldn’t disable lint checks.

Reply:
Done

Reply:
Resolving this to unblock Mohamed.